live-ingester/pom.xml

```diff
@@ -53,6 +53,22 @@
 			<artifactId>acs-event-model</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<scope>test</scope>
```

Comment:
It's probably fairly safe to put the scope of these three dependencies in the `dependencyManagement` section alongside the version.

Reply:
In the other hand in future it still will be possible to to remove the `test` scope from `dependencyManagement` section